### ISSUE TITLE
WaitForContents() macro command appears not to work

### ISSUE THREAD

**Issue by Kaner77** [Reetus/ClassicAssist#76]
Command seems to always timeout even if container contents packet is received

**Comment by Reetus** [comment]
![image](https://user-images.githubusercontent.com/6239195/99869503-6de3be80-2c06-11eb-9651-4b9c4e8bdd20.png)
Clearly not always, post your macro / shard you play on

**Comment by Kaner77** [comment]
Tried the same macro as you, still doesn't seem to work for me. I made a pull request to change the byte position of the serial number on the bag check and seemed to fix the issue. This is my first time playing around with github and looking into the code of ClassicAssist so sorry in advance if I'm making any armature mistakes :) btw I love ClassicAssist - keep up the good work!

I'm playing on CataclysmUO (http://www.cataclysmuo.com/) with ClassicUO version 0.1.6.61

![image](https://user-images.githubusercontent.com/73639797/99870188-494b0f00-2c25-11eb-83b4-56f5a82e83f9.png)
